packages/phone-number-privacy/combiner/src/common/constants.ts

```diff
@@ -1,3 +1,4 @@
 export const REASONABLE_BODY_CHAR_LIMIT: number = 2000
 export const DB_TIMEOUT = 10000
 export const MAX_BLOCK_DISCREPANCY_THRESHOLD = 3
+export const REQUEST_EXPIRY_WINDOW = 180000 // 3 minutes
```

Comment:
To make it more explicit, you can change it to:
`REQUEST_EXPIRY_WINDOW_MS = 3 * 60 * 1000`